app/Model/EstateProject.php

```diff
@@ -94,9 +94,9 @@ public function projectPhoto()
         return $this->hasOne(ProjectPhoto::class, 'project_id');
     }
 
-    private function photoDirectory()
+    public function photoDirectory()
     {
-        return Setting::PublicPath('uploads/project/');
+        return public_path('uploads/project/');
```

Comment:
The Setting::PublicPath is the right way to do this. Because there are 2 projects. 1 is the admin and the other is presentation. So this Setting object looks what is the current project and returns the path accordingly.
So you should not use public_path here.